telegram/ext/updater.py

```diff
@@ -157,6 +157,7 @@ def start_polling(self,
                 self.running = True
 
                 # Create & start threads
+                self._init_thread(self.job_queue.start, "job_queue")
```

Comment:
Here you can simply call `self.job_queue.start()`, I believe. The method is already spawning a new thread.